README.md

```diff
@@ -4,180 +4,54 @@
 
 An ID3 Decision Tree Classifier that is used to interract with the candig_server (https://github.com/CanDIG/candig-server). The classifier is implemented in such a way so that it allows differential privacy to protect personal health information (PHI).
 
-There is also a ConfusionMatrix class that extends the ID3 class which is used to determine the accuracy of the decision tree.
+There is also a ConfusionMatrix class which is used to determine the accuracy of the decision tree.
 
-Note: There is also a local implementation to be used with VCFs installed locally
-
-## Getting Started
-
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+## Quickstart
 
 ### Prerequisites
 
-What things you need to install the software and how to install them:
-```
-python2/pip (https://www.python.org/downloads/)
-virtualenv (https://virtualenv.pypa.io/en/stable/installation/)
-git (https://git-scm.com/)
-candig-server (https://github.com/CanDIG/candig-server)
-```
+- *nix environment
+- Python 3.7+
 
-### Installing ID3
-
-A step by step series of examples that tell you how to get a development env running
-
-1. Clone the repo locally
+### Building
 
+Clone the repository locally and go to the root level of the repository. Run the following command:
 ```
-git clone https://github.com/matthewwong525/ID3_project_sickkids.git
+python3 setup.py sdist bdist_wheel
 ```
 
-2. Setup virtualenv and install dependencies
+### Installing
 
+Run the following command:
 ```
-cd ID3_project_sickkids
-virtualenv id3_env
-source id3_env/bin/activate
-pip3 install -r requirements.txt
+pip install dist/id3_variants_training-0.0.0-py3-none-any.whl
```

Comment:
I'm not against this but is there a reason to prefer this two step approach (setup sdist, pip install dict) to `pip install .` ?

Reply:
Actually, now that you bring it up, there isn't really a reason. This is just the way I have been doing it whenever I want to build it.

Reply:
It's an important step for distributing the wheel (e.g. through pypi) but for getting a user up and running quickly, the simpler step is probably easier.

Reply:
So should I remove the Building section entirely and change the Installing section to pip install .? Or should I keep the Building section and just change the Installing section to pip install .?

Reply:
Maybe merge them into either "Installing" or "Setup and Install" or something like that.  You could include creating a virtualenv, which is a good idea, and would expand the setup part